docs/docs/concepts/custom_states.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Custom States
+---
+
+## Custom States
```

Comment:
@matmair it would be really helpful in the "status" documentation for each model (purchase order etc) we can link through to here. Currently this new page is not linked from anywhere else...

Reply:
Due to the newness of the feature, I do not think it should be highly advertised but I will add links